test/py/plot_pointwise_data.py

```diff
@@ -0,0 +1,132 @@
+#                 plot_pointwise_data.py
+#                 Author:  Brian Pickrell, AMD copyright 2022
+
+#   ingest the JSON files created by  test_broadcast_perf.py
+#   and print optimum local/global inputs for the test set.
+#   Use this code as an example to make pyplot charts or
+#   otherwise do follow-up analysis of the data.
+#
+#   syntax:  python3 plot_pointwise_data.py <input JSON file>
+
+from matplotlib import pyplot as plt
+import json
+import os
+import sys
+import numpy as np
+import pandas
+
+if len(sys.argv) < 2:
+    print('syntax: python3 ' + __file__ +
+          " ./test/py/pointwise/pointwise_perf_results_807270.json")
```

Comment:
These scripts should not go under test. They should go under the tools/ directory. We probably should create a new directory under tools/ to put these. Maybe tools/perf or tools/perf_analysis?